backend/handlers/invite_institution_handler.py

```diff
@@ -42,7 +41,7 @@ def post(self, user):
         if(invite.stub_institution_key):
             invite.stub_institution_key.get().addInvite(invite)
 
-        invite.send_invite(host, current_institution)
+        invite.send_invite(host, user.current_institution)
```

Comment:
Quando solicita vinculo com uma instituição:
 File "/home/mayza/workspace/web/backend/handlers/user_request_collection_handler.py", line 35, in post
    data = body['data']
KeyError: 'data'

Reply:
Continua acontecendo : 
  File "/home/mayza/workspace/web/backend/handlers/user_request_collection_handler.py", line 35, in post
    data = body['data']
KeyError: 'data'